### ISSUE TITLE
Turn off auto-aim for neutral animals and monsters and NPCs.

### ISSUE THREAD

**Issue by ifffrt** [CleverRaven/Cataclysm-DDA#48649]
# Is your feature request related to a problem? Please describe.
Something has been annoying me for a long while now and it is the fact that the auto-aim feature in this game is just so unbelievably obnoxious whenever you get a neutral NPC to follow you in your car. Whenever you want to shoot at an enemy with your turret, it automatically aims at the NPC inside your car instead of the zombie outside, and then dumbly asks "Are you sure you want to attack [NPC name]?" Every. Single. Time. Without. Fail. Imagine you have to do the healer mage quest "Kill 50 zombie" with the healer inside the car with you, and you just want to shoot at the mass grave right beside you but the game keeps wanting you to commit homicide and then have the gall to mock you for it.

Also, I don't want to kill innocent woodland animals, man. What the hell did that fox ever did to me? Yet the game automatically aimed my .50 M2 Browning anti-aircraft turret at it instead of the skeletal juggernaut in the opposite direction. And now there's one more dead cute furry animal on my conscience, right above the shorthair cat from last Thursday and the Chihuahua from last weekend. I just want to live with nature in harmony. Is that so much to ask?

# Describe the solution you'd like
Stop the killings. Make the game not automatically aim at Neutral entities anymore. Or just give us the option to turn it off for neutral creatures. (But make sure that when we manually target a neutral creature once, the game would continue to lock-on to that particular creature if we need to take another shot at it.)
# Describe alternatives you've considered
What alternative could there be but to live harmoniously with nature once again and heal the scars we've inflicted upon our beautiful planet?

# Additional context
Please.

**Comment by Salty-Panda** [comment]
What version do you use?

**Comment by ifffrt** [comment]
> 
> 
> What version do you use?

11574

**Comment by PatrikLundell** [comment]
You should definitely not remove the ability to hunt critters that are neutral, as hunting for food and other resources is a valid activity. However, it would make sense to have a priority order such that the auto targeting always targets hostile targets first and only switches to neutral ones when out of hostile ones (and place neutral ones below the hostile ones in the list). If it's possible, I'd like to see friendly characters to never be targeted automatically, but be available in the list so you can select them if you want to (the game shouldn't stop you from being a murdering mutated bastard who eats his companions, and there may be situations where you actually want to be the one to fire the first shot).

**Comment by ifffrt** [comment]
Of course we wouldn't remove the ability to hunt critters, as we could always manually target them. But you brought up something valid that I didn't think of when I made the post, which is that when auto-aim is turned off just manually targeting the neutral creature might not make auto-aim turned on again for that particular creature. So it could become a bit of a pain if we have to shoot at something repeatedly from far away. I have updated the post to reflect this.
